test/e2e/container_probe.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Probing container", func() {
 		expectNoError(err)
 		startTime := time.Now()
 
-		Expect(wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		Expect(wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
Can we make those timeouts even bigger?
If sometimes 90s is not enough, I really vote for something more like 240 than 120.
It's definitely better, to loose 30s from time to time, than to have flakes.